share/instruments/generateTs.py

```diff
@@ -4,9 +4,10 @@
 
 
 def addMessage(f, text, comment=''):
-    if (comment):
-        f.write("//: " + comment.encode('utf8') + "\n")
-    f.write('QT_TRANSLATE_NOOP("InstrumentsXML", "' + text.encode('utf8') + '"),\n')
+      if (comment):
+            f.write('QT_TRANSLATE_NOOP3("InstrumentsXML", u8"' + text.encode('utf8') + '", u8"' + comment.encode('utf8') + '"),\n')
```

Comment:
lupdate is happy with that u8 syntax?
If yes, the else case probably need it too.